### ISSUE TITLE
SmartCity-Accessibility Server Side Long Tests #16 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#353]
Build 'SmartCity-Accessibility Server Side Long Tests' is failing!

Last 50 lines of build :
```
[...truncated 5.74 KB...]
-------------------------------------------------------

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::        (v1.5.2.RELEASE)

-- Product name: JxMaps
-- Licensed version: 1.x
-- Licensed to: 
-- License type: Evaluation
-- Generation date: Dec 7, 2016
-- Expiration date: Aug 25, 2017
-- License info: Single-user license
-- Current date: May 19, 2017
JxMaps license valid.
Running smartcity.accessibility.services.LoginServiceTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 19.519 sec - in smartcity.accessibility.services.LoginServiceTest

Results :

Tests run: 1, Failures: 0, Errors: 0, Skipped: 0

[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building SmartCity-Accessibility 0.0.1-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- sonar-maven-plugin:3.3.0.603:sonar (default-cli) @ SmartCity-Accessibility ---
[INFO] User cache: /var/lib/jenkins/.sonar/cache
[ERROR] SonarQube server [http://localhost:9000] can not be reached
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 03:44 min
[INFO] Finished at: 2017-05-19T13:28:08+00:00
[INFO] Final Memory: 40M/196M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.sonarsource.scanner.maven:sonar-maven-plugin:3.3.0.603:sonar (default-cli) on project SmartCity-Accessibility: Unable to execute SonarQube: Fail to get bootstrap index from server: timeout: Socket closed -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [Alexander Kaplan] 04a9e6f3c592175fe18ef1534a32f8566baa8a22 - running client #344

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Long%20Tests/16/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
